securedrop/Dockerfile

```diff
@@ -20,6 +20,26 @@ RUN curl -LO https://launchpad.net/~ubuntu-mozilla-security/+archive/ubuntu/ppa/
     dpkg -i firefox*deb && apt-get install -f && \
     paxctl -cm /usr/lib/firefox/firefox
 
+RUN apt-get install -y wget libasound2 libdbus-glib-1-2 libgtk2.0-0 libfontconfig1 libxrender1
```

Comment:
Can you add these components to the other layer? (line 9). It's best practice to keep these things bundled. You could separate by a comment as a visual separator.